tests/decorator.py

```diff
@@ -2,7 +2,7 @@
 
 dialect_groups = {
     "duckdb": ["default"],
-    "spark": ["default"],
+    "spark": [],
```

Comment:
Sorry - this change is separate - but are you happy to merge it?  When using the test panel in vs code, the tests than run obey these settings, meaning that it will run the spark tests if I just blindly hit 'run all tests'.  

In the test panel i generally just want to run a small number of fast running tests to check everything seems ok.  I think now we run the spark tests separately in CI, removing this is safe?

Reply:
Yeah that's fine - initially just put these together to keep the previous behaviour. I generally run just the duckdb tests most of the time anyhow, so happy with this shift 👍